### PR TITLE
Fix PropertyValues sort order and update versions

### PR DESCRIPTION
**Pull request by pradh**
The main change is internal/server/v1/propertyvalues/heap.go to use "!" delimiter for comparison.  All the caches have been rebuilt with cl/466457436.

### CODE REVIEW

File: internal/server/v0/triple/golden/get_triples/stat_var.json [hunk 1]
```diff
@@ -894,7 +894,7 @@
     },
     {
       "subjectId": "Percent_Obesity_In_Count_Person",
-      "subjectName": "Population: Percentage Obesity Among",
+      "subjectName": "Population: Percentage Obese",
```
Comment (Spaceenter) [inline internal/server/v0/triple/golden/get_triples/stat_var.json:897]: Seems truncated?

File: internal/server/v1/triples/golden/triples_in/Count_Person.json [hunk 2]
```diff
@@ -499,7 +499,7 @@
           "provenanceId": "dc/d7tbsb1"
         },
         {
-          "name": "Population: Percentage Pre Obese Among",
+          "name": "Population: Percentage Pre-obese",
```
Comment (Spaceenter) [inline internal/server/v1/triples/golden/triples_in/Count_Person.json:502]: truncated?

Comment (pradh) [reply internal/server/v1/triples/golden/triples_in/Count_Person.json:502]: Intentional (cl/464971271)

File: internal/server/v0/triple/golden/get_triples/stat_var.json (hunk 1 above)
Comment (pradh) [reply internal/server/v0/triple/golden/get_triples/stat_var.json:897]: Intentional (cl/464971271)

File: internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json [hunk 3]
```diff
@@ -10503,7 +10503,7 @@
           "dcid": "geoId/0640312"
         },
         {
-          "name": "La Presa, California",
+          "name": "La Presa",
```
Comment (shifucun) [inline internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json:10506]: Due to the new ordering, the shorter name is used now:

image

Comment (pradh) [reply internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json:10506]: Interesting.

Comment (pradh) [reply internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json:10506]: Hmmm, though name string shouldn't be part of the key?  And this PR changes only the key?

Comment (shifucun) [reply internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json:10506]: The key is "dcid" + "!" + "value". When it's a leaf value, "dcid" is empty so ordering is based on value only.

Comment (pradh) [reply internal/server/v1/propertyvalues/golden/bulk_linked_property_values/result.json:10506]: Ah, I see.

`^La Presa, California^dc/prov` < `^La Presa^dc/prov`

`^La Presa!dc/prov` < `^La Presa, California!dc/prov`

**Review by shifucun — APPROVED** [review]

**Pull request merged**